tests/tools/onert_train/src/nnfw_util.cc

```diff
@@ -49,63 +50,50 @@ uint64_t bufsize_for(const nnfw_tensorinfo *ti)
   return elmsize[ti->dtype] * num_elems(ti);
 }
 
+std::string toString(const NNFW_TRAIN_OPTIMIZER &opt)
+{
+  static const std::unordered_map<NNFW_TRAIN_OPTIMIZER, std::string> name_map{
+    {NNFW_TRAIN_OPTIMIZER_UNDEFINED, "undefined"},
+    {NNFW_TRAIN_OPTIMIZER_ADAM, "adam"},
+    {NNFW_TRAIN_OPTIMIZER_SGD, "sgd"},
+  };
+  return name_map.at(opt);
```

Comment:
(Q) How does it work if it receives a value that is not defined in enum in the `toString` method?

Reply:
It might throw an error in line `name_map.at()` line.